### DISCUSSION TITLE
When will the next version arrive

### DISCUSSION THREAD

**Discussion by wyhghost** [bpatrik/pigallery2#962] [General]
Thank you very much for creating the pigallery,It perfectly solved my problem, and I really like it.
May I ask when the next version will be updated? It has been over a year since the last update.
May everything be well with you :)

**Comment by bpatrik** [comment]
Hi,

Unfortunately I got very busy recently so I cannot tell a date. It won't be
out in the next 4 months for sure.

**Reply by wyhghost** [reply]
How have you been lately? Looking forward to the new version.
I wish you all well.

**Comment by DiDe2304** [comment]
Any updates about a new version?

**Comment by skatsubo** [comment]
Hey @bpatrik, hopefully all is well. Thank you so much for the awwwesome project!
Recently we got several nice PRs/improvements ready to merge. Please let us know if/how we can assist. Would be happy to help if you are busy these days.

**Comment by bpatrik** [comment]
Hi,

I'm on a Holliday right now. I shall take a look at the PRs once I'm back
home

-- Sorry for being brief, sent from my phone.

**Comment by bpatrik** [comment]
so generally about the timeline, i try to collect all bugs and planned features under milestone. 

This is the next one: https://github.com/bpatrik/pigallery2/milestone/9

All (well most of it) needs to be fixed for the next release to come out. If anyone sees an issue and would like to give s tab to it, I'm happy to accept PRs

I plan 2-3 major changes for the next release:
1. extension support (minimal backend related addons already done and can be tested on the :edge) #743
2. creating gallery projections based on a search querry. Basically slicing your gallery and limiting all the features based on a search query
    * this will allow sharing searches and also adding allow/balclist to a user (like hiding keyword:private photos from your parents, while still showing to your wife) #1015
3. dng support.
